tests/bluetooth/bsim/audio/src/tbs_test.c

```diff
@@ -25,8 +37,8 @@ static void tbs_hold_call_cb(struct bt_conn *conn, uint8_t call_index)
 static bool tbs_originate_call_cb(struct bt_conn *conn, uint8_t call_index,
 				  const char *caller_id)
 {
-	printk("Placing call to remote with id %u to %s\n",
-	       call_index, caller_id);
+	printk("Placing call to remote with id %u to %s\n", call_index,
+	       caller_id);
```

Comment:
Might as well do if you intend on keeping this change: The maximum line length in Zephyr is 100 (and the 80 recommendation have been removed).

```suggestion
	printk("Placing call to remote with id %u to %s\n", call_index, caller_id);
```